src/hvac.rs

```diff
@@ -0,0 +1,112 @@
+use std::net::Ipv4Addr;
+
+use packed_struct::PackedStructSlice;
+use phf::phf_map;
+
+use crate::{
+    constants,
+    network::{
+        util::reverse_mac, AirCondInfo, AirCondState, DiscoveryResponse, HvacDataCommand,
+        HvacDataMessage,
+    },
+    Device, DeviceInfo,
+};
+
+/// A mapping of hvac device codes to their friendly model equivalent.
+pub const HVAC_CODES: phf::Map<u16, &'static str> = phf_map! {
+    0x4E2Au16 => "Licensed manufacturer",
```

Comment:
Is this the actual friendly name of the device? If it is, then all is good, but it seems weird that it would present itself as "Licensed manufacturer".

Reply:
The code should work with many AC manufacturers which is based on this device code (and probably is using `ACFreedom` application to control).
I am not very familiar with this market but AFAICT most of them are probably similar models but branded for specific manufacturers. This is why I called it like this (in fact I found similar method in another code):
https://github.com/mjg59/python-broadlink/pull/520/files#diff-a29a838665bc485155f9126b5a9c8616d588def9ca2468095d112f58e8650c35R108

Moreover some of the devices returns chinese characters in name (but this is not always the case). Anyway I think that it is valuable information, so the library is showing it like this (my case):
```
奥克斯空调 [HVAC "Licensed manufacturer"] (address = 192.168.0.89, mac = <REDACTED>, locked? = false)
```